docs(website): improve docs around csp and cloud regions (#1346)
commercetools/merchant-center-application-kit: website/src/content/deployment/runtime-configuration.mdx

}
```

## Production environment

<Warning>

When the application is deployed on a production environment, the application will fail to make requests to the [API Gateway](/main-concepts/api-gateway). This is due to the fact that the domain, where the application is hosted, is not whitelisted by the CORS rules of the API Gateway.

Therefore, it's important that the application is [registered](/register-applications/configuring-a-custom-application), runs behind the proxy router, and that the `servedByProxy` option in the `env.json` is set to `true`.

</Warning>

> It is recommended to have two separate configuration files, one for development and one for production. The `env.json` **must** be used for development, therefore for production you can have a `env.prod.json` file.

Reviewer: ```suggestion
It is recommended to have two separate configuration files, one for development and one for production. The `env.json` **must** be used for development, therefore for production you can have an `env.prod.json` file.
```

# `headers.json`

> This configuration file allows to configure HTTP headers. The most important one is the [Content Security Policy](https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP) header.

Reviewer: ```suggestion
This configuration file allows you to configure HTTP headers. The most important one is the [Content Security Policy](https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP) header.
```

Reviewer 2: @adinakleine  the you is not necessary and I am relatively sure that it's not our typical style until now, too.

<Info>

For development the `headers.json` file is **optional**. If used, the file must be in the root path of the project as it's automatically loaded by the Webpack dev server.

</Info>

## Content Security Policy

> The Content Security Policy (CSP) header allows to instruct the browser how to deal with security measures, according to the configured instructions. Most of the times it's enough to whitelist the hostnames that the Custom Application uses, for example if [static assets](/deployment/serving-static-assets) are served by a CDN.

Reviewer: ```suggestion
The Content Security Policy (CSP) header allows you to instruct the browser how to deal with security measures, according to the configured instructions. Most of the time it's enough to whitelist the hostnames that the Custom Application uses, for example if [static assets](/deployment/serving-static-assets) are served by a CDN.
```

<Info>

Since Custom Applications are hosted on any domain of your choice, it's **imperative** that any hostname used by the application is whitelisted in the CSP configuration.
Furthermore, the [API Gateway](/main-concepts/api-gateway#hostnames) hostnames **must** also be listed in the CSP configuration.

</Info>

For example, given that your application is hosted at `my-app.now.sh` and runs on the [Google Cloud Europe region](/main-concepts/api-gateway#cloud-regions), the `headers.json` file must be configured as following:

```json
{
  "csp": {
    "script-src": ["my-app.now.sh"],
    "connect-src": [

>       "my-app.now.sh",

Reviewer 2: If we propose this as a copy-paste example including the specific "my-app.now.sh"  I suggest to actually reserve this now.sh  domain in our account (it's free) to avoid potential future attempts to hijack custom applications that were created from our boilerplate, installed but never finished.  

I could be a static 200 reponse on everyting or something like that (e.g. just a now.json file)

(not necessary to merge this PR, just an idea)

Author: Hmm I see your point but I wouldn't worry much about it. The name `my-app` is very generic and I don't expect people to use that.
This example is also not really meant to be a copy-paste. The deployments examples are more likely to do that.
